drivers/timer/arcv2_timer0.c

```diff
@@ -49,10 +49,12 @@
 
 #define TICKLESS (IS_ENABLED(CONFIG_TICKLESS_KERNEL))
 
+#define SMP_TIMER_DRIVER (CONFIG_SMP && CONFIG_MP_NUM_CPUS > 1)
+
```

Comment:
See earlier, insn't CONFIG_SMP enough? SMP with 1 CPU does not really make sense.